src/redis-benchmark.c

```diff
@@ -1617,7 +1617,14 @@ int main(int argc, const char **argv) {
     argc -= i;
     argv += i;
 
+    slot_placeholder = zmalloc(1);
+    slot_placeholder[0] = '\0';
+
     if (config.cluster_mode) {
+        // We only include the slot placeholder {tag} if cluster mode is enabled
+        slot_placeholder = zrealloc(slot_placeholder,7);
+        slot_placeholder = ":{tag}";
```

Comment:
the second line here just overrides the allocation with a static string pointer.
i don't care about the leak in this case, but there's really no need for any zmalloc / zrealloc in this case.
just init the static pointer with `""`, and later override it with `":{tag}"`.
while we're at it, i think a shorter name for the variable would have been nicer, maybe just `tag` (which is by default empty)?

Reply:
addressed comments in the latest commit. Thank you Oran :+1: